scripts/include.sh/build-dep.sh

```diff
@@ -155,14 +155,15 @@ build_git_ios()
 
 build_git_osx()
 {
-  sdk="`xcodebuild -showsdks 2>/dev/null | grep macosx | sed 's/.*macosx\(.*\)/\1/'`"
-  archs="x86_64"
+xc
+  sdk="`xcodebuild -showsdks 2>/dev/null | grep '-sdk macosx' | sed 's/.*macosx\(.*\)/\1/'`"
```

Comment:
I don't think this is safe, this output might change at any time.
You are going to be much more resilient when you are using something like the json output using
```json
$ xcodebuild -showsdks -json
....
  {
    "buildID" : "2DA5241A-FE0E-11EA-BEF2-093A02C66744",
    "canonicalName" : "appletvos14.2",
    "displayName" : "tvOS 14.2",
    "isBaseSdk" : true,
    "platform" : "appletvos",
    "platformPath" : "/Applications/Xcode-beta.app/Contents/Developer/Platforms/AppleTVOS.platform",
    "platformVersion" : "14.2",
    "productBuildVersion" : "18K5036d",
    "productCopyright" : "1983-2020 Apple Inc.",
    "productName" : "Apple TVOS",
    "productVersion" : "14.2",
    "sdkPath" : "/Applications/Xcode-beta.app/Contents/Developer/Platforms/AppleTVOS.platform/Developer/SDKs/AppleTVOS14.2.sdk",
    "sdkVersion" : "14.2"
  }
....
```

One example where this can go wrong (and probably does go wrong looking at this code) is that Apple added DriverKit with the identifier `driverkit.macosx20`.

So the current and new command does the following:

```sh
$ xcodebuild -showsdks 2>/dev/null | grep macosx 
	DriverKit 20.0                	-sdk driverkit.macosx20.0
	macOS 11.0                    	-sdk macosx11.0
```

Which is probably not what you want.

If you are ok depending on tools like that can deal with structured data like json one option would be to filter the output for the **product name** and not the identifier.

e.g.
```sh
$ xcodebuild -showsdks -json  | jq -r -c '.[] | select(.productName == "macOS") | .canonicalName'
macosx11.0
```